app/services/chat.ts

```diff
@@ -221,6 +238,13 @@ export class ChatService extends Service {
 
     const settings = this.customizationService.state;
 
+    new GuestApiHandler().exposeApi(this.chatView.webContents.id, {
+      pinMessage: (messageData: IChatHighlightMessage) =>
+        this.chatHighlightService.pinMessage(messageData),
+      unpinMessage: () => this.chatHighlightService.unpinMessage(),
+      showUnpinButton: () => Promise.resolve(this.chatHighlightService.hasPinnedMessage.getValue()),
+    });
```

Comment:
In the interest of minimizing privileged surface area where it's not needed, can we only expose the API if `hasChatHighlightWidget` is true?